src/pr-info.ts

```diff
@@ -399,6 +399,34 @@ configSuspicious["OTHER_FILES.txt"] = contents =>
     // all path parts are not empty and not all dots
     : !contents.split(/\n/).every(line => line === "" || isRelativePath(line)) ? "bad path"
     : undefined;
+configSuspicious["package.json"] = (contents, oldText) => {
```

Comment:
Done :heavy_check_mark:
- I've applied the same changes to #194 and #196
- I used a `for...of` loop in place of `forEach()` because I prefer the new syntax to the library function, but I'll switch it back if you prefer.
- I used a `function` statement vs. assignment to define `diffFromReq()` but likewise, I'll switch it back if you prefer.

Reply:
I indeed prefer `forEach` since I find it more readable, and I prefer a `const` definition since I'm coming from a FP background.  (BTW, it's not an assignment, it's a definition -- very important difference...)

In any case, don't bother about changing anything: since the same pattern of "either some shape or converging to that shape" pattern is repeated, I will abstract it out later to make the code more readable...

Reply:
My bad, sounds good! I'll leave the code alone but I'll rebase it, assuming you haven't already started rebasing this PR and that would be helpful?

Reply:
Bah, I see that it's *just* too late --- I'd warn you against spending time since more rebases are likely once more PRs are merged... :)